### PR TITLE
adjusting the demos so that overriding size and color works again

### PR DESCRIPTION
**Pull request by dlockhart**

### CODE REVIEW

File: demo/icons/icon-size-override.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+class IconSizeOverride extends LitElement {
+
+	static get styles() {
+		return css`
+			:host {
+				display: inline-block;
+			}
+			::slotted(d2l-icon) {
+				--d2l-icon-height: 100px;
+				--d2l-icon-width: 100px;
```
Comment (dlockhart) [inline demo/icons/icon-size-override.js]: So this doesn't work in IE11 only... Firefox, Safari, Edge and Chrome it's fine. Uh... any ideas?

Comment (dbatiste) [reply demo/icons/icon-size-override.js]: Interesting.  I wonder if shadyCSS is scoping those variables to the IconSizeOverride local DOM scope?  Or, does it make any difference if those assignments are done inside of `:host` instead?

File: demo/icons/icon.html [hunk 2]
```diff
@@ -49,25 +42,25 @@ <h3>Custom non-SVG (src attribute)</h3>
...
-			<div class="size-override">
+			<d2l-icon-demo-size-override>
 				<d2l-icon icon="d2l-tier3:calendar"></d2l-icon>
 				<d2l-icon src="../../components/icons/images/tier3/calendar.svg" size="tier3"></d2l-icon>
 				<d2l-icon src="calendar-tier3.png" size="tier3"></d2l-icon>
-			</div>
+			</d2l-icon-demo-size-override>
 		</d2l-demo-snippet>
 
 		<h3>RTL</h3>
```
Comment (dbatiste) [inline demo/icons/icon.html]: It seems like the RTL specific snippet is somewhat redundant now.  Perhaps we just choose an icon for the basic icon snippet that happens to be RTL sensitive?

File: demo/icons/icon-size-override.js (hunk 1 above)
Comment (dlockhart) [reply demo/icons/icon-size-override.js]: I tried that as well, also doesn't work. When I inspect the CSS that's getting inserted into the `<head>`, there's nothing there.

File: demo/icons/icon.html (hunk 2 above)
Comment (dlockhart) [reply demo/icons/icon.html]: Yeah I was thinking about that as well. Either that or just rely on the future visual diffs to test this.

Comment (dlockhart) [reply demo/icons/icon.html]: Updated to use a different icon that's RTL and removed this section.

**Pull request merged**